dlp/snippets/inspect/inspect_bigquery.go

```diff
@@ -156,7 +156,7 @@ func inspectBigquery(w io.Writer, projectID string, infoTypeNames []string, cust
 	// Create the inspect job.
 	j, err := client.CreateDlpJob(ctx, req)
 	if err != nil {
-		return fmt.Errorf("CreateDlpJob: %w", err)
+		return fmt.Errorf("CreateDlpJob: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.